spec/bolt/application_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'bolt/application'
+require 'bolt/plan_creator'
```

Comment:
We should probably just move the `require 'bolt/plan_creator'` in `Bolt::CLI` to `Bolt::Application` so it will already by loaded.